src/javascript/app/pages/user/new_account/financial_acc_opening.js

```diff
@@ -11,16 +11,26 @@ const State          = require('../../../../_common/storage').State;
 
 const FinancialAccOpening = (() => {
     const form_id = '#financial-form';
+    const client_details = sessionStorage.getItem('client_form_response');
```

Comment:
I think this needs to be in the onload function since it needs to happen when this page loads, right?

Reply:
Actually, it would be the same since we just need to get this details once before it's required, so I moved it onLoad to encapsulate the variable scope.